content/en/network_monitoring/devices/troubleshooting.md

```diff
@@ -12,91 +12,165 @@ further_reading:
 
-#### Can I still use Network Device Monitoring if my device-model pair isn't supported?
+   ```
+   Error: check device reachable: failed: decryption error; failed to autodetect profile: failed to fetch sysobjectid: cannot get sysobjectid: decryption error; failed to fetch values: failed to fetch scalar oids with batching: failed to fetch scalar oids: fetch scalar: error getting oids `[1.3.6.1.2.1.1.1.0 1.3.6.1.2.1.1.2.0 1.3.6.1.2.1.1.3.0 1.3.6.1.2.1.1.5.0]`: decryption error  OR Error: check device reachable: failed: wrong digest; failed to autodetect profile: failed to fetch sysobjectid: cannot get sysobjectid: wrong digest; failed to fetch values: failed to fetch scalar oids with batching: failed to fetch scalar oids: fetch scalar: error getting oids `[1.3.6.1.2.1.1.1.0 1.3.6.1.2.1.1.2.0 1.3.6.1.2.1.1.3.0 1.3.6.1.2.1.1.5.0]`: wrong digest
+   ```
 
-Datadog collects generic base-line metrics from all devices. If there are unsupported metrics from a vendor MIB, you can write a custom profile, or send a feature request to [Datadog support][1].
+   **Solution**: 
+   Ensure the following configuration parameters are correct: `user`, `authKey`, `authProtocol`, `privKey`, `privProtocol`.
 
-If you send a feature request, Datadog support needs a `snmpwalk` from the requested device. Run the following and send the output:
+### Traps not being received for devices
 
-```
-snmpwalk -O bentU -v 2c -c <COMMUNITY_STRING> <IP_ADDRESS>:<PORT> 1.3.6
-```
+1. Check the Datadog `agent.log` file to ensure that you aren't unable to bind to the traps port. If this is the case, the following error is present:
 
-#### Why am I only seeing one metric collected for my networks and it's the number of devices collected at zero?
+   ```
+   Failed to start snmp-traps server: error happened when listening for SNMP Traps: listen udp 0.0.0.0:162: bind: permission denied
+   ```
 
-1. Try loosening ACLs/firewall rules for your devices.
-2. Run `snmpwalk -O bentU -v 2c -c <COMMUNITY_STRING> <IP_ADDRESS>:<PORT> 1.3.6` from the host your Agent is running on. If you get a timeout without any response, there is likely something blocking the Datadog Agent from collecting metrics from your device.
+   **Solution**:
+   Add a net bind capability to the Agent binary, which allows the Agent to bind to reserved ports: 
 
-#### What do I do if Datadog supports a vendor or device type but my specific model isn't supported? 
+   ```
+   sudo setcap 'cap_net_bind_service=+ep' /opt/datadog-agent/bin/agent/agent
+   ```
 
-- Contact [Datadog support][1] to put in a request to support your specific model.
-- Extend your profiles to support additional `sysobjectid` values. 
-    For example, if you want to monitor another type of Cisco CSR, you can modify the ISR profile directly to list another `sysobjectid` like this: 
-    
-    ```
-		snmpwalk -v 2c -c [community string] [ip address] 1.3.6.1.2.1.1.2
-    ```
+2. Navigate to the troubleshooting dashboard in NDM:
```

Comment:
Wait, when do I move on to step 2? There was a solution in step 1, what if that solved my problem? Should I keep going?

Reply:
ahh yes that is confusing, will fix !